app/grandchallenge/components/serializers.py

```diff
@@ -1,57 +1,152 @@
 from rest_framework import serializers
 
-from grandchallenge.cases.models import Image
+from grandchallenge.cases.models import Image, RawImageUploadSession
 from grandchallenge.components.models import (
     ComponentInterface,
     ComponentInterfaceValue,
+    InterfaceKind,
 )
+from grandchallenge.core.validators import JSONSchemaValidator
+from grandchallenge.reader_studies.models import ANSWER_TYPE_SCHEMA
 
 
 class ComponentInterfaceSerializer(serializers.ModelSerializer):
     kind = serializers.CharField(source="get_kind_display", read_only=True)
 
     class Meta:
         model = ComponentInterface
-        fields = [
-            "title",
-            "description",
-            "slug",
-            "kind",
-            "pk",
-        ]
+        fields = ["title", "description", "slug", "kind", "pk"]
 
 
 class SimpleImageSerializer(serializers.ModelSerializer):
     # Used for component interface values where only the user provided
     # name is needed
     class Meta:
         model = Image
-        fields = (
-            "pk",
-            "name",
-        )
+        fields = ("pk", "name")
 
 
-class ComponentInterfaceValueSerializer(serializers.ModelSerializer):
-    # Serializes images in place rather than with hyperlinks for internal usage
-    image = SimpleImageSerializer()
-    interface = ComponentInterfaceSerializer()
+class ComponentInterfaceValuePostSerializer(serializers.ModelSerializer):
+    """
+    Serializes images with hyperlinks for external usage
+    Expects interface_title, to allow creating a ComponentInterfaceValue with an
+    existing ComponentInterface
+    """
+
+    image = serializers.HyperlinkedRelatedField(
+        queryset=Image.objects.all(),
+        view_name="api:image-detail",
+        required=False,
+    )
+
+    interface_title = serializers.CharField(write_only=True)
+    interface = ComponentInterfaceSerializer(read_only=True)
+    upload_pk = serializers.CharField(write_only=True, required=False)
 
     class Meta:
         model = ComponentInterfaceValue
         fields = [
+            "interface_title",
             "interface",
             "value",
             "file",
             "image",
             "pk",
+            "upload_pk",
         ]
 
+    def validate(self, attrs):  # noqa: C901
+        interface = ComponentInterface.objects.get(
+            title=attrs["interface_title"]
+        )
+        attrs.pop("interface_title")
+        attrs["interface_id"] = interface.id
```

Comment:
`_pk` and `.pk` is preferred, just in case you somehow end up dealing with a non-ORM object (it happens...)